src/Countries/Tunisia.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Calendars\HijriCalendar;
+
+class Tunisia extends Country
+{
+    use HijriCalendar;
+
+    /**
+     * @return string
+     */
```

Comment:
No need for these docblocks when the type is already specified. Only array structures should be typed.